fdbserver/Status.actor.cpp

```diff
@@ -2328,12 +2359,14 @@ ACTOR Future<StatusReply> clusterGetStatus(
 			state Future<ErrorOr<vector<std::pair<MasterProxyInterface, EventMap>>>> proxyFuture = errorOr(getProxiesAndMetrics(db, address_workers));
 
 			state int minReplicasRemaining = -1;
+			state Optional<Value> primaryDCO;
 			std::vector<Future<JsonBuilderObject>> futures2;
 			futures2.push_back(dataStatusFetcher(ddWorker, configuration.get(), &minReplicasRemaining));
 			futures2.push_back(workloadStatusFetcher(db, workers, mWorker, rkWorker, &qos, &data_overlay, &status_incomplete_reasons, storageServerFuture));
 			futures2.push_back(layerStatusFetcher(cx, &messages, &status_incomplete_reasons));
 			futures2.push_back(lockedStatusFetcher(db, &messages, &status_incomplete_reasons));
 			futures2.push_back(clusterSummaryStatisticsFetcher(pMetrics, storageServerFuture, tLogFuture, &status_incomplete_reasons));
+			futures2.push_back(getActivePrimaryDC(cx, &primaryDCO, &messages));
```

Comment:
Oh, my apologies. I didn't notice that this futures vector wasn't using something like `success` to make them all `Void`. I guess the cleanest way to do this would be to not put the future into the vector but instead just start it up here at the same time as the others and then wait for it when you need it down below. Then you wouldn't have to use the dummy return value.